baselines/dasha/dasha/client.py

```diff
@@ -0,0 +1,363 @@
+"""Defines Flower Clients."""
```

Comment:
Many methods are set as `_method_name` making them not publich (and therefore not required by the formatting/tests checks to include some minimal comment of what it does). Could you please add some comments ? The idea is if someone else wants to run/understand your code they should know a bit what's going on, for instance, in `_stochastic_compression_step()`, what's `mega_batch_size`, etc. If you think it's useful to add a ref to subsections/equations in the paper, then that's cool too.

Reply:
Ok, done!

Reply:
thanks!!